### PR TITLE
Ids and replace fixex

### PR DESCRIPTION
**Pull request by gabrielbori**
- Added global to replace function when setting accesibility labels. 
- Added accesibility label in movie preview
- Added accesibility label in explore card.

### CODE REVIEW

File: src/components/movie/MovieInlinePreview.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ class MovieInlinePreview extends React.Component<Props> {
     const { movie } = this.props;
     const { title, year, vote_average, poster_path } = movie;
     return (
-      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replace(' ', '_')}  >
+      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replace(/ /g, '_')}  >
```
Comment (marianozicavo) [inline src/components/movie/MovieInlinePreview.tsx:41]: You can do this as well
```suggestion
      <TouchableHighlightView scaleFactor={0.98} contentStyle={styles.container} onPress={this.onInlinePreviewPress} accessibilityLabel={title.replaceAll(' ', '_')}  >
```

File: src/components/movie/MoviePreview.tsx [hunk 2]
```diff
@@ -72,7 +72,7 @@ class MoviePreview extends React.PureComponent<Props> {
   render() {
     const { movieId } = this.props;
     return (
-      <TouchableScale disabled={!movieId} scaleFactor={0.97} style={styles.container} onPress={this.onPress}>
+      <TouchableScale disabled={!movieId} scaleFactor={0.97} style={styles.container} onPress={this.onPress} accessibilityLabel={movieTitle}>
```
Comment (marianozicavo) [inline src/components/movie/MoviePreview.tsx]:
```suggestion
      <TouchableScale disabled={!movieId} scaleFactor={0.97} style={styles.container} onPress={this.onPress} accessibilityLabel={movie.original_title}>
```

File: src/components/movie/MoviePreview.tsx [hunk 3]
```diff
@@ -56,8 +56,8 @@ class MoviePreview extends React.PureComponent<Props> {
 
   renderMoviePreview() {
     const { movie, highPriority, withRatingBadge } = this.props;
+    movieTitle=movie.original_title;
```
Comment (marianozicavo) [inline src/components/movie/MoviePreview.tsx]:
```suggestion
```

File: src/components/movie/MoviePreview.tsx [hunk 4]
```diff
@@ -19,7 +19,7 @@ import { AppText, TouchableScale } from '../common';
 const { width } = Dimensions.get('window');
 const previewWidth = width * 0.27;
 export const getMoviePreviewHeight = () => previewWidth / theme.specifications.posterAspectRation;
-
+var movieTitle;
```
Comment (marianozicavo) [inline src/components/movie/MoviePreview.tsx]:
```suggestion
```

File: src/components/movie/SectionHorizontalScroll.tsx [hunk 5]
```diff
@@ -36,7 +36,7 @@ class SectionHorizontalScroll extends React.PureComponent<Props> {
     const { title } = sectionData[sectionKey];
 
     return (
-      <View style={styles.container} accessibilityLabel={title.replace(' ','_')+"_section"}>
+      <View style={styles.container} accessibilityLabel={title.replace(/ /g,'_')+"_section"}>
```
Comment (marianozicavo) [inline src/components/movie/SectionHorizontalScroll.tsx:39]:
```suggestion
      <View style={styles.container} accessibilityLabel={title.replaceAll(' ','_')+"_section"}>
```

**Review by marianozicavo — APPROVED** [review]

File: src/components/movie/MovieInlinePreview.tsx (hunk 1 above)
Comment (gabrielbori) [reply src/components/movie/MovieInlinePreview.tsx:41]: Apparently RN does not support "replaceAll" function. We leave it as it is.

File: src/components/movie/SectionHorizontalScroll.tsx (hunk 5 above)
Comment (gabrielbori) [reply src/components/movie/SectionHorizontalScroll.tsx:39]: Apparently RN does not support "replaceAll" function. We leave it as it is.

File: src/components/movie/MoviePreview.tsx (hunk 2 above)
Comment (gabrielbori) [reply src/components/movie/MoviePreview.tsx]: fixed

**Review by marianozicavo — APPROVED** [review]

**Pull request merged**